charts/kubewarden-controller/templates/audit-scanner.yaml

```diff
@@ -53,14 +59,6 @@ spec:
             nodeSelector:
               {{- toYaml . | nindent 14 }}
             {{- end }}
```

Comment:
Not related with this PR, but `nodeSelector` doesn't seem to be part of [`Container`](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.30/#container-v1-core).

Am I missing something? This shouldn't work at all unless we move it to the level above.

Reply:
Good catch! I'll fix it